modules/ROOT/pages/index.adoc

```diff
@@ -219,55 +311,16 @@ include::partial$common.adoc[tag="windows-separators-warning"]
 --
 
 ====
-+
+
 [sidebar]
 ====
 For information about adding an ODBC data source, see the https://support.microsoft.com/en-us/office/administer-odbc-data-sources-b19f856b-5b9b-48c9-8b93-07484bfab5a7#bm2[Microsoft support documentation].
 ====
 
-== Install the Couchbase Power BI Connector
-You will need to install the connector that allows Power BI to communicate with Couchbase, using ODBC.
-
-. Download `couchbase-powerbi-connector-1.0.mez` from https://packages.couchbase.com/releases/couchbase-powerbi-connector/1.0/couchbase-powerbi-connector-1.0.mez[] --
-see the xref:release-notes.adoc[release notes page] for more details.
-
-. Follow https://learn.microsoft.com/en-us/power-bi/connect-data/desktop-connector-extensibility#custom-connectors[Microsoft's guide] on configuring a custom connector.
-
-
-=== Verify the Connector is Loaded Successfully:
-
-. Open *Power BI*.
-
-. Go to *Get Data -> More -> Couchbase Connector*
-
-
-The Couchbase Power BI Connector is now correctly configured to work with the Power BI platform.
-
-Next, you configure a connection to Couchbase Analytics or an instance of the Capella columnar database.
-
-
-== Configure an ODBC Data Source
-
-You configure a data source name (DSN) for each CBAS scope or Capella columnar database you want to use in Power BI. 
-
-
-
-=== Configure a User DSN in Power BI
-
-You can install either *Couchbase ODBC Driver (ANSI)* or *Couchbase Unicode Driver (UNICODE)*. 
-If the data contains multibyte characters, select Unicode.
-
-[IMPORTANT]
-Be careful to use the *ODBC Data Source Administrator (64 bit)*; there is an *ODBC Data Source Administrator (32 bit)*
-available, but this will not work with the connector.
-
-When the *Create data source for Couchbase* dialog opens,
-select *Capella Columnar* or (for CBAS) *Couchbase Analytics* and enter values as follows: 
-
 
 == Use the Couchbase Power BI Connector
 
-Business information tools rely on data that is organized into relational databases. 
+Business information tools rely on data organized into relational databases.
```

Comment:
This is related to line no 336: When prompted for *Username* and *Password*, supply your Capella columnar or CBAS credentials. 
This should be access account in columnar, database access in case of Capella Provisioned. These creds are identical to ODBC DSN Username and Password fields in all three cases.
https://preview.docs-test.couchbase.com/rays-preview/power-bi-connector/current/index.html#configure-an-odbc-data-source